demo/src/examples/4 Origin and Destination.svelte

```diff
@@ -19,7 +19,7 @@
       style,
       center: [-74.1197632, 40.6974034],
       zoom: 11,
-      customAttribution: "<a href='http://project-osrm.org/' target='_blank'>&copy; OSRM</a>",
+      customAttribution: "<a href='//project-osrm.org/' target='_blank'>&copy; OSRM</a>",
```

Comment:
Interesting, can you comment what this does? Is it because they don't have https?

Reply:
No, that's just because my IDE is unhappy with "HTTP" in links. And double slash is a universal one meaning use the same protocol as this page does.